src/server/gql/schema/__tests__/Email.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { PrismaClient } from '@prisma/client'
+import { ApolloServer, ExpressContext, gql } from 'apollo-server-express'
+import { mockReset } from 'jest-mock-extended'
+import { DeepMockProxy } from 'jest-mock-extended/lib/mjs/Mock'
+import { GraphQLResponse } from 'apollo-server-types'
+import { GraphQLError } from 'graphql'
+import { sign } from '../../../jwt'
+import startApolloServer from '../../apolloServer'
+
+import prisma from '../../prisma'
+
+jest.mock('../../prisma.ts')
+
+describe('Email schema', () => {
+  let apollo: ApolloServer
+  let prismaMock: DeepMockProxy<PrismaClient>
+
+  beforeEach(async () => {
+    mockReset(prisma)
+    prismaMock = prisma as unknown as DeepMockProxy<PrismaClient>
```

Comment:
**Question**

Instead of using `__mocks__/prisma.ts`, which will require us to perform these type assertions in every test, could we have a `mockPrisma` file we import directly? We have this in CVP I believe.

Reply:
I gave it a shot, just as CVP, for whatever reason the mocking was not working. Actual Prisma was being called. 

Let's address this later.